bin/check-es-index-field-count.rb

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env ruby
+#
+#   check-es-index-field-count
+#
+# DESCRIPTION:
+#   This plugin checks if the number of fields in ES index(es) is approaching limit. ES by default
+#   puts this limit at 1000 fields per index.
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: aws_es_transport
```

Comment:
As this is clearly not being maintained anymore I think we should not include it, we are gonna rip that out from the checks in the somewhat near future.
https://github.com/cdunn/aws-es-transport/issues/3